### PR TITLE
Adding admin dashboard (#12)

### PR DESCRIPTION
**Pull request by ahmed-fawzy-metwally**

### CODE REVIEW

File: js/admin.js [hunk 1]
```diff
@@ -0,0 +1,1019 @@
...
+        }
+    }
+    return false;
+
+    // oldCategories.forEach(element => {
+    //     if(element.name === categoryName){
+    //         return true;
+    //     }
+    // });
+    // return false;
```
Comment (Hossam-Tarek) [inline js/admin.js]: Did you forget to uncomment those lines, or they were for testing? If those lines are for testing your code please remove them.

File: js/admin.js [hunk 2]
```diff
@@ -0,0 +1,1019 @@
...
+        });
+    }());
+
+}
+
+function editProducts(productId) {
+
+    // Add Notification
+
+}
```
Comment (Hossam-Tarek) [inline js/admin.js:723]: Why this function is empty remove it if you'll not use it

**Comment by Hossam-Tarek** [comment]
An amazing effort and respect for that. Well done

File: js/admin.js (hunk 1 above)
Comment (ahmed-fawzy-metwally) [reply js/admin.js]: No, I wanna ask our instructor why this block of code didn't work correctly.
but you have a right, in the final version all unuseful comments must be removed.
Do you do it? or I update it before you merge

File: js/admin.js (hunk 2 above)
Comment (ahmed-fawzy-metwally) [reply js/admin.js:723]: If we have time we gonna handle it. As we have decided

File: js/admin.js (hunk 1 above)
Comment (Hossam-Tarek) [reply js/admin.js]: Remove before merge

File: js/admin.js (hunk 2 above)
Comment (Hossam-Tarek) [reply js/admin.js:723]: ok

File: js/admin.js (hunk 1 above)
Comment (ahmed-fawzy-metwally) [reply js/admin.js]: Done

**Comment by Hossam-Tarek** [comment]
Greate work

**Comment by Hossam-Tarek** [comment]
Great work

**Pull request merged**